examples/argocd-with-applications/apps/argocd.yaml

```diff
@@ -0,0 +1,85 @@
+"apiVersion": "argoproj.io/v1alpha1"
```

Comment:
Can you remove the apps folder? this is required so that there are no collisions in new installations

Reply:
Successfully removed.